schemas/telemetry/pioneer-study/pioneer-study.4.schema.json

```diff
@@ -0,0 +1,75 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "properties": {
+    "creationDate": {
+      "pattern": "^[0-9]{4}-[0-9]{2}-[0-9]{2}T[0-9]{2}:[0-9]{2}:[0-9]{2}\\.[0-9]{3}Z$",
+      "type": "string"
+    },
+    "id": {
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+      "type": "string"
+    },
+    "payload": {
+      "description": "",
+      "properties": {
+        "encryptedData": {
+          "description": "JOSE/JWE encrypted payload.",
+          "type": "string"
+        },
+        "encryptionKeyId": {
+          "description": "JOSE/JWK key id, e.g. pioneer-20170520.",
+          "type": "string"
+        },
+        "pioneerId": {
+          "description": "Custom pioneer id, must not be Telemetry clientId",
+          "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
+          "type": "string"
+        },
+        "studyName": {
+          "description": "Name of a particular study. Usually the addon_id.",
+          "maxLength": 100,
+          "minLength": 1,
+          "pattern": "^\\S+$",
+          "type": "string"
+        },
+        "studyVersion": {
```

Comment:
I missed one.  If this is the version we are using for the nested validation schema it should be an integer.

Reply:
Changed to integer version for the schema. I copied the semantic version approach from the shield study schema.

Reply:
Yeah that is the exact problem: you are using the last shoe-horned 'need it now' hack as the basis for new development.  A few months from now someone will say we are just doing what pioneer does and try to further propagate the technical debt and processing overhead.